spec/support/fake_objects.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'cucumber/core/filter'
```

Comment:
Traditionally in spec support files you would use contexts and then allow extra contexts with let blocks described.

However given this is a large piece of work you've done, maybe I can build ontop of this as my first contribution back in a while?

Reply:
Yes, I agree. I was pretty focus in trying to fix offenses without disabling cop.

That allow to show us more explicitly where good refactoring topics are.

I suggest to refactor as part of other PRs